frontend/ui/widget/container/widgetcontainer.lua

```diff
@@ -55,6 +55,7 @@ function WidgetContainer:paintTo(bb, x, y)
         self.dimen = Geom:new{x = 0, y = 0, w = content_size.w, h = content_size.h}
     end
 
+    -- NOTE: Clunky or left in on the off-chance we're passed a dimen that isn't a proper Geom object...
```

Comment:
(Spent 30 seconds reading this :) I guess (right?) a comma between `in` and `on` would have reduced it to 3s :)

Reply:
Yeah, I should have backticked the `or` to make it clearer ;).